.circleci/config.yml

```diff
@@ -774,13 +774,15 @@ jobs:
           name: pre-release docker images
           command: |
             BRANCH=${CIRCLE_BRANCH}
-            docker/build_push.sh -u -p -b ${BRANCH} -n client
-            docker/build_push.sh -u -p -b ${BRANCH} -n init
-            docker/build_push.sh -u -p -b ${BRANCH} -n faucet
-            docker/build_push.sh -u -p -b ${BRANCH} -n tools
-            docker/build_push.sh -u -p -b ${BRANCH} -n validator
-            docker/build_push.sh -u -p -b ${BRANCH} -n validator-tcb
-            docker/build_push.sh -u -p -b ${BRANCH} -n cluster-test
+            success=0
+            docker/build_push.sh -u -p -b ${BRANCH} -n client || success=-1
+            docker/build_push.sh -u -p -b ${BRANCH} -n init || success=-1
+            docker/build_push.sh -u -p -b ${BRANCH} -n faucet || success=-1
+            docker/build_push.sh -u -p -b ${BRANCH} -n tools || success=-1
+            docker/build_push.sh -u -p -b ${BRANCH} -n validator || success=-1
+            docker/build_push.sh -u -p -b ${BRANCH} -n validator-tcb || success=-1
+            docker/build_push.sh -u -p -b ${BRANCH} -n cluster-test || success=-1
```

Comment:
shouldn't we evaluate success before performing these operations?

Reply:
Image signing is failing in this script, but it can still push an unsigned image. The point is to attempt all images so we at least get unsigned images rather than no images.

Reply:
if the command fails, we set success to -1 so it can be reported at the end of all the steps in the flow.   Several things may have gone wrong.   maybe the image couldn't be built, maybe it couldn't be signed (the problem this worked around) and maybe it couldn't be pushed to dockerhub.   Regardless if they could be built we want to continue and build them all (in the current case they could all be built and pushed to dockerhub but docker signing appears to be down down/unreachable).    Errors will still be reported at the end of the build to allow for investigation.